torch/_inductor/autotune_process.py

```diff
@@ -662,11 +664,15 @@ def make_run_fn(
             from torch._C import _cuda_getCurrentRawStream as get_raw_stream
 
             stream = get_raw_stream(self.output_tensor_meta.device.index)
-
+        if self.workspace_arg is not None:
+            # Triton will generate code with the arg signature (args, output, workspace)
+            # We thus need to flip the order of the last entry in input_tensor and output_tensor
+            arg_list = list(input_tensors[:-1]) + [output_tensor, input_tensors[-1]]
+        else:
+            arg_list = input_tensors + [output_tensor]
```

Comment:
Okay after learning some more about how this all works, I think it would be better if we can just generate the workspace on a bmrq by bmrq basis since the checking code quickly gets squirely when you have extern kernels, aten, and triton.